api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -160,8 +160,7 @@ type AzureManagedControlPlaneSpec struct {
 	LoadBalancerSKU *string `json:"loadBalancerSKU,omitempty"`
 
 	// IdentityRef is a reference to a AzureClusterIdentity to be used when reconciling this cluster
-	// +optional
-	IdentityRef *corev1.ObjectReference `json:"identityRef,omitempty"`
+	IdentityRef *corev1.ObjectReference `json:"identityRef"`
```

Comment:
/hold
Should we make the same change for AzureCluster?

Reply:
For AzureCluster the type is on the clusterclass and it's already enforced at a per cluster level with https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/api/v1beta1/azurecluster_validation.go#L145

Reply:
Makes sense!
/hold cancel

Reply:
/hold

Would it be better to make it required everywhere and add the webhook validation to AzureManagedControlPlane as well for consistency?